tools/shared/src/main/scala/org/scalajs/core/tools/javascript/ScalaJSClassEmitter.scala

```diff
@@ -558,6 +558,10 @@ final class ScalaJSClassEmitter(semantics: Semantics, outputMode: OutputMode,
     val isAncestorOfHijackedClass =
       AncestorsOfHijackedClasses.contains(className)
 
+    val isRawJSType =
+      if (kind == ClassKind.RawJSType) js.BooleanLiteral(true)
+      else js.Undefined()
```

Comment:
Because trailing `undefined`s are pruned away in `prunedParams`, and they're ordered carefully to maximize the number of trailing `undefined`s.